tap_stripe/__init__.py

```diff
@@ -725,6 +714,56 @@ def sync_sub_stream(sub_stream_name, parent_obj, updates=False):
                 Context.new_counts[sub_stream_name] += 1
 
 
+def sync_sub_stream(sub_stream_name, bookmark_value):
+    """
+    Get the parent records based on the bookmark and corresponding child stream records
+    """
+    stream_name = PARENT_STREAM_MAP[sub_stream_name]
```

Comment:
Don't use stream, use child_stream or parent_stream for better readability.

Reply:
Updated the code